### DISCUSSION TITLE
Скрытие вывода пустых веток дерева

### DISCUSSION THREAD

**Discussion by juramebel** [totumonline/totum-ru-issues-and-discussions#152] [Есть идея, а что если...]
Здравствуйте!
Как сделать так, чтобы не загружались ветки дерева, в которых нет содержимого?
Сейчас когда загружаю таблицу, в которой дерево по другой таблице, отображаются все его ветви, даже в которых пусто.

**Comment by ttm-alexeypolunin** [comment]
Сделаем параметр для Селект-дерева и `tree`.

**Reply by erpic-ru** [reply]
Апну, чтобы не потерялось. Для кастомного поиска иногда нужно.

**Reply by ttm-alexeypolunin** [reply]
Когда-нибудь
